### DISCUSSION TITLE
Hide by default

### DISCUSSION THREAD

**Discussion by emavitta** [vuejs/petite-vue#85] [Show and tell]
I encountered a little problem which i solved, but i'm open to better suggestion.
When there are elements that have to be hidden by default with a v-if statement, they are visibile until the script load.
To bypass this problem i used on a tag
`:style="{display:show}" style="display:none;"`
and as a data variable
`show:'block',`

this way, once the script is loaded it set the display as block but also hide it with the v-if condition.

Better suggestion?

**Comment by MartinMystikJonas** [comment]
I think you are looking for v-cloak https://v3.vuejs.org/api/directives.html#v-cloak

**Reply by emavitta** [reply]
And i think you are right.... thanks!

**Comment by emavitta** [comment]
It's working but i had to move the v-cloak attribute to a different tag then the one with also the v-if...
